analyzers/maven/maven.go

```diff
@@ -1,6 +1,7 @@
 // Package maven implements Maven analysis.
 //
-// A `BuildTarget` for Maven is the Maven project name.
+// A `BuildTarget` for Maven is either a path to a directory containing a POM manifest or a path to such a
```

Comment:
I should re-phrase that; it's either:
1. a path to a directory in which there is a "pom.xml" file; or
2. a path to a "pom.xml" file

Actually I'll need to update that to say that it can also be a `groupId:artifactId` project identifier.